app/src/main/AndroidManifest.xml

```diff
@@ -29,14 +36,19 @@
         </activity>
 
         <activity
-            android:name=".MainActivity"
+            android:name=".Login"
```

Comment:
When name change happens for Login class, change the activity name here as well.